semgrep/tests/e2e/test_ci.py

```diff
@@ -2,10 +2,8 @@
 import re
 import shutil
 import subprocess
-import sys
 from pathlib import Path
 from textwrap import dedent
-from unittest import mock
```

Comment:
I installed `pytest-mock` so instead of ever importing mock, we should use the `mocker` fixture instead.